docs/examples/driver_examples/Qcodes example with AMI430.ipynb

```diff
@@ -638,23 +638,75 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Use the ``i3d.pause`` method to request all three axes instruments to pause ramping:"
+    "**NOTE** that ramping with the ``\"simultaneous\"`` changes the ramp rates of the individual magnet axes. The implementation is such that, given the current and the setpoint magnetic field vectors together with either vector ramp rate (``vector_ramp_rate`` parameter) or the desired ramp duration (``ramp_simultaneously`` method), the required ramp rates of the individual axes are calculated and then set via the ``ramp_rate`` parameters of the individual magnet axes. Once the individual ramp rates are set, the ramping is initiated on all axes (almost) at the same time. However, the values of the individual ramp rates can be **restored** by calling ``wait_while_all_axes_ramping``. Note that if ``block_during_ramp`` parameter is set to ``True``, ``wait_while_all_axes_ramping`` method is called automatically by the ramping logic, thus the ramp rates of the individual magnet axes are **restored** after the ramp is finished. If you'd like to use the simultanous ramp with ``block_during_ramp`` parameter set to ``False`` **AND** you'd like the ramp rates of the individual magnet axes to be restored, just call ``wait_while_all_axes_ramping`` at the appropriate moment in your measurement code, see the following example:\n",
```

Comment:
I would try to refrase this to always encurage users to call ``wait_while_all_axes_ramping`` after doing a non blocking ramp. Something like `this will ensure that the magnet is in a known good state before doing the next operation``

Reply:
done